lib/trino-orc/src/main/java/io/trino/orc/writer/DictionaryBuilder.java

```diff
@@ -72,6 +72,11 @@ public DictionaryBuilder(int expectedSize)
         this.containsNullElement = false;
     }
 
+    public static boolean isNullIndex(int index)
```

Comment:
nit: move above constructor per code style (`static public methods above instance methods`)

Reply:
@sopel39 Thank you for your review. However, I'm trying to close the PR as I haven't seen any significant performance gains, as in the conversation above.